src/manager_vmcore_task_form.xhtml

```diff
@@ -10,4 +10,5 @@
         <input type="submit" value="Create vmcore task" class="submit" />
         <div class="center">Any URL that <code>wget</code> can download or a local path (<code>file:///foo/bar</code> or just <code>/foo/bar</code>)</div>
       </div>
+      <input type="checkbox" {md5_enabled} name="md5sum" class="checkbox" /><span class="formsubhead"> Calculate md5 checksum for all downloaded resources</span>
```

Comment:
It will be better if there will be "label" tag instead of "span"
   https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_label
But it that would require change everywhere else in form too. So for this PR this should be ok.